bundle/manifests/rhods-operator.clusterserviceversion.yaml

```diff
@@ -52,6 +52,9 @@ metadata:
               "trainingoperator": {
                 "managementState": "Removed"
               },
+              "trustyai": {
+                "managementState": "Removed"
```

Comment:
Should the value be set to Managed? Will the component GA in 2.12?

Reply:
it will only get into RHOAI as  TP first, no plan when set to GA for now.